src/player.cpp

```diff
@@ -4657,8 +4657,8 @@ void player::update_stamina( int turns )
             stamina_recovery += bonus;
         }
     }
-
-    stamina = roll_remainder( stamina + stamina_recovery * turns );
+    //  recover ratio was balanced for legacy 6s/turn, hence the division by 6
+    stamina = roll_remainder( stamina + stamina_recovery * turns / 6 );
```

Comment:
```suggestion
    stamina = roll_remainder( stamina + stamina_recovery * turns / 6.0f );
```
I would do this just to make things very clear that this is not integer division.